sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/models/TopicDescription.java

```diff
@@ -635,7 +714,7 @@ public Boolean isEnableSubscriptionPartitioning() {
      * @param enableSubscriptionPartitioning the enableSubscriptionPartitioning value to set.
      * @return the TopicDescription object itself.
      */
-    public TopicDescription setEnableSubscriptionPartitioning(Boolean enableSubscriptionPartitioning) {
+    TopicDescription setEnableSubscriptionPartitioning(Boolean enableSubscriptionPartitioning) {
         this.enableSubscriptionPartitioning = enableSubscriptionPartitioning;
```

Comment:
should this API be called `enableSubscriptionPartitioning`

Reply:
This is package-private, so it doesn't matter.